pkg/apis/controller/v1beta1/schedulingspec.go

```diff
@@ -45,17 +45,46 @@ type SchedulingSpec struct {
 	Spec SchedulingSpecTemplate `json:"spec,omitempty" protobuf:"bytes,1,rep,name=spec"`
 }
 
+type ClusterReference struct {
+	Name string `json:"name"`
+}
+
+type ClusterSchedulingSpec struct {
+	Clusters        []ClusterReference    `json:"clusters,omitempty"`
+	ClusterSelector *metav1.LabelSelector `json:"clusterSelector,omitempty"`
+}
+
+type ScheduleTimeSpec struct {
+	Min     metav1.Time `json:"minTimestamp,omitempty"`
+	Desired metav1.Time `json:"desiredTimestamp,omitempty"`
+	Max     metav1.Time `json:"maxTimestamp,omitempty"`
+}
+
+type DispatchDurationSpec struct {
+	Expected int `json:"expected,omitempty"`
+	Limit    int `json:"limit,omitempty"`
+	Overrun  bool
+}
+
+type DispatchingWindowSpec struct {
+	Start ScheduleTimeSpec `json:"start,omitempty"`
+	End   ScheduleTimeSpec `json:"end,omitempty"`
+}
+
 type SchedulingSpecTemplate struct {
-	NodeSelector map[string]string `json:"nodeSelector,omitempty" protobuf:"bytes,1,rep,name=nodeSelector"`
-	MinAvailable int               `json:"minAvailable,omitempty" protobuf:"bytes,2,rep,name=minAvailable"`
+	NodeSelector      map[string]string     `json:"nodeSelector,omitempty" protobuf:"bytes,1,rep,name=nodeSelector"`
+	MinAvailable      int                   `json:"minAvailable,omitempty" protobuf:"bytes,2,rep,name=minAvailable"`
+	ClusterScheduling ClusterSchedulingSpec `json:"clusterScheduling,omitempty"`
+	DispatchingWindow DispatchingWindowSpec `json:"dispatchingWindow,omitempty"`
+	DispatchDuration  DispatchDurationSpec  `json:"dispatchDuration,omitempty"`
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 type SchedulingSpecList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata"`
 
-	Items []SchedulingSpec `json:"items"`
+	Items []SchedulingSpec `json:"SchedulingSpecTemplate"`
```

Comment:
Out of curiosity, this has been there all along as `json:"items"` what happens with this change?